packages/client/hmi-client/src/components/workflow/ops/compare-datasets/tera-compare-datasets-drilldown.vue

```diff
@@ -106,6 +106,22 @@
 				header="Output settings"
 				content-width="360px"
 			>
+				<template #overlay>
+					<tera-chart-settings-panel
+						:annotations="
+							[ChartSettingType.VARIABLE, ChartSettingType.VARIABLE_COMPARISON].includes(
+								activeChartSettings?.type as ChartSettingType
+							)
+								? getChartAnnotationsByChartId(activeChartSettings?.id ?? '')
+								: undefined
+						"
+						:active-settings="activeChartSettings"
+						:generate-annotation="generateAnnotation"
```

Comment:
Since you are using own custom chart options and variables in `useCompareDatasetCharts`, you probably need to provide your own function for generate-annotation as well to include correct variables, options and the translationMap, other wise, llm won't give good results since it can't recognize the variables names you used for the chart.

Reply:
I'll leave this to another pr (sorry I am doing this again)

Reply:
I'll leave this to another pr (sorry I am doing this again)